src/components/errorPage/content/emailChangeVerification.tsx

```diff
@@ -0,0 +1,27 @@
+import { Trans } from '@smg-automotive/i18n-pkg';
+import React from 'react';
+import Box from 'src/components/box';
+
+const EmailChangeVerificationErrorContent = () => {
+  return (
+    <Box width="full">
+      <p>
+        <Trans i18nKey="errorPage.EMAIL_CHANGE_VERIFICATION_ERROR.content.email">
+          <strong></strong>
+        </Trans>
+      </p>
+      <p>
```

Comment:
I tried to dedupe the formatting but it resulted in some split error of the transl component, did not digg any deeper

![Screenshot 2024-10-18 at 12 18 13](https://github.com/user-attachments/assets/a68bcf45-837c-45b7-b576-fa6a303e9465)